bl-languages/fi_FI.json

```diff
@@ -0,0 +1,242 @@
+{
+	"language-data":
+	{
+		"native": "Suomi (Suomi)",
+		"english-name": "Finnish",
+		"last-update": "2016-06-11",
+		"author": "Tuomas K.",
+		"email": "",
+		"website": ""
+	},
+
+	"username": "Käyttäjätunnus",
+	"password": "Salasana",
+	"confirm-password": "Vahvista salasana",
+	"editor": "Kirjoittaja",
+	"dashboard": "Hallintapaneeli",
+	"role": "Tyyppi",
+	"post": "Blogikirjoitus",
+	"posts": "Blogikirjoitukset",
+	"users": "Käyttäjät",
+	"administrator": "Ylläpitäjä",
+	"add": "Lisää",
+	"cancel": "Peruuta",
+	"content": "Sisältö",
+	"title": "Otsikko",
+	"no-parent": "Ei ole",
+	"edit-page": "Muokkaa sivua",
+	"edit-post": "Muokkaa blogikirjoitusta",
+	"add-a-new-user": "Lisää uusi käyttäjä",
+	"parent": "Onko tämä jonkun sivun alasivu?",
+	"friendly-url": "Sivun osoite",
+	"description": "Kuvaus",
+	"posted-by": "Julkaisija",
+	"tags": "Tagit",
+	"position": "Sijainti navigaatiossa",
+	"save": "Tallenna",
+	"draft": "Luonnos",
+	"delete": "Poista",
+	"registered": "Käyttäjä luotu",
+	"notifications": "Ilmoitukset",
+	"profile": "Profiili",
+	"email": "Sähköpostiosoite",
+	"settings": "Asetukset",
+	"general": "Yleiset",
+	"advanced": "Lisäasetukset",
+	"regional": "Alueellinen",
+	"about": "Tietoja",
+	"login": "Kirjaudu sisään",
+	"logout": "Kirjaudu ulos",
+	"manage": "Hallitse",
+	"themes": "Teemat",
+	"prev-page": "Edellinen sivu",
+	"next-page": "Seuraava sivu",
+	"configure-plugin": "Lisäosan asetukset",
+	"confirm-delete-this-action-cannot-be-undone": "Haluatko varmasti poistaa? Tätä toimintoa ei voi perua.",
```

Comment:
"confirm-delete-this-action-cannot-be-undone": "Vahvistetaanko poistaminen? Tätä toimintoa ei voi perua.",